style.css

```diff
@@ -0,0 +1,72 @@
+.container-nav {
+  display: flex;
+  height: auto;
+  justify-content: space-between;
+}
+
+.logo-name {
+  font-family: 'Poppins', sans-serif;
+  font-size: 25px;
+  font-weight: bold;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #6070ff;
+  letter-spacing: 5px;
+  margin-left: 10px;
+}
+
+.navlog {
+  size: 25px;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 10px;
+}
+
+.container {
+  background-image: url(/img/background.png);
+}
+
+.title {
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  color: #172b4d;
+  align-items: center;
+  margin: 26px;
+}
+
+.intro {
+  font-family: 'Poppins', sans-serif;
+  font-weight: bold;
+  font-size: 16px;
+  margin: 10px;
+  text-align: justify;
+}
+
+.connect {
+  font-family: 'Poppins', sans-serif;
+  font-weight: bolder;
+  text-transform: uppercase;
+  font-size: 18px;
+  word-spacing: 3px;
+  color: #7f8cff;
+  margin-left: 24px;
+  margin-top: 28px;
+  margin-bottom: 10px;
+}
+
+.networks {
+  text-decoration: none;
+  margin: none;
+  color: #505f79;
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 24px;
+  padding-top: 10px;
+  gap: 12px;
+}
+
+.linkedin {
+  color: solid #505f79;
+}
```

Comment:
- [ ] [REQUIRED] Good start! In the `header` section, give the right `color` value to the elements, it is a little too `light`, pay attention to the `position` of the main `heading` and `paragraph` and give the desired style, as shown in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=8kOjLtGM8zmLoMSW-0), and pay attention to the `color` of the `social icons` some of them are not in the required color. Finally, the `background image` is not available at the bottom of the figure web page, so please `style` it as shown in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=8kOjLtGM8zmLoMSW-0) 🙏🏼 🙏🏼 I believe you will do it easily and I wish you to approve in the next review 😃 

![image](https://user-images.githubusercontent.com/63915024/219444561-b47967f0-5181-4797-9f4e-a6568633dc8f.png)

Reply:
@mukhammadrizooff in the logo and the linkedin logo i dont know what its happening cause the color its the one that is on figma, do you know what could be doing that?